agent/setup.go

```diff
@@ -296,7 +296,6 @@ func getPrometheusDefs(cfg lib.TelemetryConfig) ([]prometheus.GaugeDefinition, [
 		HTTPSummaries,
 		consul.ACLSummaries,
 		consul.ACLEndpointSummaries,
-		consul.ACLEndpointLegacySummaries,
```

Comment:
If you hadn't already, this should go on the short list of stuff that needs cleaned up on the website page for metrics.

Reply:
Oh, good call. I can probably add that to this PR